iFixit/react-commerce
# Add footer

Closes #76 

## Changes

- Added GraphQL codegen to autogenerate typescript types from Strapi GraphQL (stuff under `lib/api/strapi/generated` does not need to be CR since it's auto generated by graphql codegen)
- Added footer components
- Refactored layout component to account for footer data requirements
- Refactored collection folder to get data from Strapi

## QA

1. Visit the Vercel PR preview
2. Visit a collection page and verify that the footer renders correctly
3. Verify that the footer data comes from the us store settings in Strapi
